src/Microsoft.ML.Data/Transforms/Hashing.cs

```diff
@@ -126,17 +126,17 @@ public sealed class ColumnInfo
             /// <summary>
             /// Describes how the transformer handles one column pair.
             /// </summary>
-            /// <param name="input">Name of input column.</param>
-            /// <param name="output">Name of the column resulting from the transformation of <paramref name="input"/>. Null means <paramref name="input"/> is replaced.</param>
+            /// <param name="source">Name of column to work on.</param>
+            /// <param name="name">Name of the column resulting from the transformation of <paramref name="source"/>. Null means <paramref name="source"/> is replaced.</param>
```

Comment:
>Null means <paramref name="source"/> is replaced [](start = 118, length = 48)

Other way around. `null` on *source* means this. Also FYI, while I'm pleased the see that the parameters themselves are in the right order, the `params` tags themselves are not reversed.